smart-contracts/wasm-transform/src/artifact.rs

```diff
@@ -252,6 +255,7 @@ impl TryFromImport for ArtifactNamedImport {
     fn ty(&self) -> &FunctionType { &self.ty }
 }
 
+/// An iterator over local variables.
 pub struct LocalsIterator<'a> {
     /// Number of locals that are still going to be yielded from the iterator.
     remaining_items:      u32,
```

Comment:
What is meant by the `Current multiplicity of the current_item` in the comment below?

Reply:
Docs missing `LocalsIterator::new` a few lines below.

Reply:
Added

Reply:
And elaborated on the current multiplicity.